spark/src/main/java/org/apache/iceberg/actions/ExpireSnapshotsAction.java

```diff
@@ -90,6 +91,19 @@ protected Table table() {
     return table;
   }
 
+  /**
+   * Whether or not to use stream the expired file list to the driver. The default (false) will use
+   * collect to bring back all results to the driver at once which may be an issue with very long file lists.
```

Comment:
What about this? `By default, all files to delete are brought to the driver at once which may be an issue with very long file lists. Set this to true to use {@link Dataset#toLocalIterator()} if you are running into memory issues when collecting the list of files to be deleted.`

Reply:
Sgtm

Reply:
Sounds good to me, except that it creates a dead link because `Dataset#toLocalIterator` is not in Iceberg Javadoc.